doc/api/n-api.md

```diff
@@ -697,6 +697,58 @@ functions) that can be passed as parameters when `js_callback` is invoked. This
 pointer is managed entirely by the threads and this callback. Thus this callback
 should free the data.
 
+#### napi_key_collection_mode
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+todo
+
+```C
+typedef enum {
+  napi_key_include_prototypes,
+  napi_key_include_own_properties
+} napi_key_collection_mode;
+```
+
+#### napi_key_filter
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+todo
+
+```C
+typedef enum {
+  napi_key_all_properties = 0,
+  napi_key_writable = 1,
+  napi_key_enumerable = 1 << 1,
+  napi_key_configurable = 1 << 2,
+  napi_key_skip_strings = 1 << 3,
+  napi_key_skip_symbols = 1 << 4
+} napi_key_filter;
+```
+
+#### napi_key_conversion
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+todo
```

Comment:
There is still a TODO note.

Reply:
Sorry If I've been not clear 😅. What I mean is remove the `todo` notes, but not capitalize them. Please replace them with anything meaningful to make the PR self-contained.